tests/sentry/api/endpoints/test_organization_release_health_data.py

```diff
@@ -1658,6 +1661,8 @@ def test_derived_metric_incorrectly_defined_as_singular_entity(
         mocked_get_mri_query.return_value = "crash_free_fake"
         mocked_reverse_mri.return_value = "crash_free_fake"
         mocked_parse_mri.return_value = ParsedMRI("e", "sessions", "crash_free_fake", "none")
+        mocked_parse_field.return_value = MetricField(None, "e:sessions/crashed_free_fake@none")
```

Comment:
This has to be mocked so that it passes newly added validation, but triggers error in other part of code, which was original intention. This metric is fully mocked and it does not exists, it was only created to test specific use case